util/execute-deploy.sh

```diff
@@ -50,6 +51,12 @@ deploy_library \
   dagger.pom.xml
 
 deploy_library \
+  java/dagger/libgwt.jar \
+  java/dagger/libgwt.jar \
```

Comment:
They don't have sources, so using the same target here avoids building unnecessary things: all we need is some non-empty JAR to deploy (see note in http://central.sonatype.org/pages/requirements.html#supply-javadoc-and-sources), and because the `:gwt` JAR is already really small we can just reuse it.